src/functionalTest/resources/features/F-009/F-009.feature

```diff
@@ -0,0 +1,103 @@
+@F-009
+Feature: F-009 : Post Role Assignments Advance Query Request
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-183
+  Scenario: must successfully receive Role Assignments without specific page number
+    Given a user with [an active IDAM profile with full permissions],
```

Comment:
This is just a new version of the existing API..so we need to check only those features which are added.